features/nanostack/FEATURE_NANOSTACK/mbed-mesh-api/source/include/static_config.h

```diff
@@ -145,6 +145,13 @@ extern "C" {
 #define MBED_MESH_API_THREAD_CONFIG_NETWORK_NAME "Thread Network"
 #endif
 
+// is static config used?
+#if defined MBED_CONF_MBED_MESH_API_THREAD_STATIC_LINK_CONFIG
+#define MBED_MESH_API_THREAD_STATIC_LINK_CONFIG MBED_CONF_MBED_MESH_API_THREAD_STATIC_LINK_CONFIG
+#else
+#define MBED_MESH_API_THREAD_STATIC_LINK_CONFIG 0X1
+#endif
```

Comment:
No.
You already have defined `MBED_CONF_MBED_MESH_API_THREAD_STATIC_LINK_CONFIG` use it.
Don't redefine extra macro names.